integrations/tensorflow-keras/Neptune-TensorFlow-Keras.ipynb

```diff
@@ -0,0 +1,681 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "G4ny3mHCFrqG",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "# Neptune + TensorFlow / Keras"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Before we start"
```

Comment:
Maybe this should drop this in showcase/docs?

Now everything from install to init neptune falls under this category.